tests/subsys/pm/power_mgmt/src/dummy_driver.c

```diff
@@ -9,33 +9,88 @@
 #include <pm/device_runtime.h>
 #include "dummy_driver.h"
 
+static bool dont_sleep;
+extern bool z_sys_post_kernel;
+
 static int dummy_open(const struct device *dev)
 {
-	return pm_device_get(dev);
+	dont_sleep = false;
+	pm_device_get(dev);
+	if (dev->pm->enable == false) {
+		pm_device_enable(dev);
+		return pm_device_get(dev);
+
+	}
+	return 0;
```

Comment:
Not getting all this complexity, if this function is showing how to use runtime pm, it should just call pm_device_get()

Reply:
yes, updated.